### DISCUSSION TITLE
Welcome to rh-test Discussions!

### DISCUSSION THREAD

**Discussion by guanruihua** [guanruihua/rh-test#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by huaAlpha** [comment]
添加多几种 使用测试用例的方法, 现在主要都是使用test方法

**Comment by guanruihua** [comment]
测试用例书写太麻烦, 添加方法, 生成响应的测试用例的方法
